utils/debug/api.go

```diff
@@ -53,6 +54,7 @@ func GetAccessAPIBlockHeader(
 		return nil, err
 	}
 
+	//nolint:structwrite
```

Comment:
I don't think it's a good idea to shortcut it, even for debug purposes lets just provide a correct value.